### DISCUSSION TITLE
멘토링 회고 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#94] [Ideas]
생각해보니 멘토링 받은 내용에 대해서도 회고를 진행하면 좋을 것 같다는 생각이 들어서..

멘토님도 진행하시는데 저도 동일하게 코멘트에 제 생각이랑 내용을 정리해볼까 합니다..!

**Comment by jongfeel** [comment]
그러면 좋죠.
